testsuite/thermo_integration/test_thermo_integration.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import espressopp
+import mpi4py.MPI as MPI
+
+import unittest
+
+class TestThermoIntegration(unittest.TestCase):
+    def setUp(self):
+
```

Comment:
@acfogarty Catch me ;) 
@fedepad yes, I meant about the blank line after the method name